### ISSUE TITLE
A fatal error has been detected by the Java Runtime Environment

### ISSUE THREAD

**Issue by qty1994** [Homebrew/homebrew-core#150965]
### `brew gist-logs <formula>` link OR `brew config` AND `brew doctor` output

```shell
HOMEBREW_VERSION: 4.1.15
ORIGIN: https://github.com/Homebrew/brew
HEAD: e57a87cfe9c15f192cd4fc6695a2b3b7acf0ddf7
Last commit: 6 days ago
Core tap JSON: 15 Oct 00:58 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit icelake
Clang: 15.0.0 build 1500
Git: 2.39.3 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 8.1.2 => /usr/bin/curl
macOS: 14.0-x86_64
CLT: 15.0.0.0.1.1694021235
Xcode: N/A

Your system is ready to brew.
```

### Verification

- [X] My "`brew doctor` output" says `Your system is ready to brew.` and am still able to reproduce my issue.
- [X] I ran `brew update` and am still able to reproduce my issue.
- [X] I have resolved all warnings from `brew doctor` and that did not fix my problem.
- [X] I searched for recent similar issues at https://github.com/Homebrew/homebrew-core/issues?q=is%3Aissue and found no duplicates.

### What were you trying to do (and why)?

homebrew install jmeter

### What happened (include all command output)?

A fatal error has been detected by the Java Runtime Environment:
#
#  SIGILL (0x4) at pc=0x00007ff80d317b73, pid=8190, tid=259
#
# JRE version: OpenJDK Runtime Environment Homebrew (17.0.8.1) (build 17.0.8.1+0)
# Java VM: OpenJDK 64-Bit Server VM Homebrew (17.0.8.1+0, mixed mode, sharing, tiered, compressed oops, compressed class ptrs, g1 gc, bsd-amd64)
# Problematic frame:
# C  [AppKit+0xc8fb73]  _NSCarbonMenuCrashIfNeeded+0x258
#
# No core dump will be written. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
# An error report file with more information is saved as:
# /Users/qty/hs_err_pid8190.log
#
# If you would like to submit a bug report, please visit:
#   https://github.com/Homebrew/homebrew-core/issues
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#
/usr/local/Cellar/jmeter/5.6.2/libexec/bin/jmeter: line 199:  8190 Abort trap: 6           "$JAVA_HOME/bin/java" $ARGS $JVM_ARGS $JMETER_OPTS -jar "$PRGDIR/ApacheJMeter.jar" "$@"
[hs_err_pid8190.log](https://github.com/Homebrew/homebrew-core/files/12908374/hs_err_pid8190.log)

### What did you expect to happen?

service ran successfully

### Step-by-step reproduction instructions (by running `brew` commands)

```shell
brew install jmeter
jmeter
A fatal error has been detected by the Java Runtime Environment:
```

**Comment by SMillerDev** [comment]
Duplicate of https://github.com/Homebrew/homebrew-core/issues/150824
